independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
+    "version": "1.0",
+     "contact": {
+      "name": "Kiveshan Naidoo, Rishay Sunder",
+      "email": "kiveshannaidoo9@gmail.com"
+    }
+  },
+  "host": "opentdb.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.